src/components/accordion/_accordion.scss

```diff
@@ -70,15 +74,24 @@
     // This is styled to look like a link not a button
     .govuk-accordion__open-all {
       @include govuk-font($size: 16);
-      @include govuk-link-common;
-      display: inline;
+      position: relative;
+      z-index: 1;
+      margin: 0;
```

Comment:
If you don't do this the open all button focus state renders below the border

Reply:
Sorry my comment refers to `margin: 0`, not the `z-index`. I still can't see setting `margin: 0` making any difference. But this is non-blocking

Reply:
I think it's just being explicit rather than relying on the user agent styles. The open all button is just using the default user agent padding at the minute, for example. Even if most (all?) browsers use `margin: 0` it's still good to be clear that that's intentional.

Reply:
👍